### PR TITLE
Removed unused variables from microphysics methods

### PR DESCRIPTION
**Pull request by fmalatino**
**Description**
Removal of `d0_vap` and `lv00` from `sedimentation` method calls and `cracw` from `icloud` method calls in `microphysics.py` as they were causing parsing issues

**How Has This Been Tested?**
Tested using currently implemented translate tests in `pySHiELD` and in `dace:cpu` backend runs in `pace`.

**Checklist:**
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules
- [x] New check tests, if applicable, are included
- [x] Targeted model, if this change was triggered by a model need/shortcoming

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -12,6 +12,7 @@
     "f90nml>=1.1.0",
     "numpy",
     "xarray",
+    "types-pyyaml",
```
Comment (FlorianDeconinck) [inline setup.py]: Not needed here if you put it in the pre-commit additional dependencies

**Review by FlorianDeconinck — DISMISSED** [review]
LGTM, just a double dependency to clear out

File: setup.py (hunk 1 above)
Comment (fmalatino) [reply setup.py]: Removed.

**Review by oelbert — APPROVED** [review]
Looks good. We should clean up some of the field/constant confusion here, but the new microphysics is coming soon™ anyway, so it's not a huge priority

**Pull request merged**